Utilities.cs

```diff
@@ -0,0 +1,236 @@
+            ProductsList.Add(p1);
+            ProductsList.Add(p2);
+            ProductsList.Add(p3);
+            inventory = new Inventory(ProductsList);
+        }
+
+        internal static void ShowMainMenu()
+        {
+            Console.ResetColor();
+            Console.Clear();
+            Console.WriteLine("********************");
+            Console.WriteLine("* Select an action *");
+            Console.WriteLine("********************");
+
+            Console.WriteLine("1: Add Product");
+            Console.WriteLine("2: View all products");
+            Console.WriteLine("3: Edit a product");
+            Console.WriteLine("4: Delete a product");
+            Console.WriteLine("5: Search for a product");
+            Console.WriteLine("0: Close application");
+
+            Console.Write("Your selection: ");
+
+            string? userSelection = Console.ReadLine();
+            switch (userSelection)
+            {
+                case "1":
+                    AddProduct();
+                    break;
+                case "2":
+                    ViewAllProducts();
+                    break;
+                case "3":
+                    EditProductOption();
+                    break;
+                case "4":
+                    DeleteProductOption();
+                    break;
+                case "5":
+                    SearchForProductOption();
+                    break;
+                case "0":
+                    break;
+                default:
+                    Console.WriteLine("Invalid selection. Please try again.");
+                    break;
+            }
+
+        }
+        private static void AddProduct()
+        {
+            Console.Write("Enter Product Name: ");
+            string? ProductName = Console.ReadLine();
+
+            Console.Write("Enter Product Price: ");
+            double ProductItemPrice = double.Parse(Console.ReadLine());
+
+            Console.Write("Enter Product Currency: ");
+            string? ProductItemCurrency = Console.ReadLine().ToUpper();
+
+            Console.Write("Enter Product Quantity: ");
+            int ProductQuantity = int.Parse(Console.ReadLine());
+
+            Currency CurrencyValue;
+            Enum.TryParse(ProductItemCurrency, out CurrencyValue);
+
+            inventory.AddProductToInventory(new Product(ProductName, new Price() { ItemPrice = ProductItemPrice, Currency = CurrencyValue }, ProductQuantity));
+
+            AddProductSubMenu();
+        }
+        private static void AddProductSubMenu()
+        {
+            Console.WriteLine("* What do you want to do more? *");
+            Console.WriteLine("********************");
+            Console.WriteLine("1: Add Other Product");
+            Console.WriteLine("2: Go Back to Main Menu");
+            Console.WriteLine("0: Close application");
+            Console.Write("Your selection: ");
+            string Nextopetion = Console.ReadLine();
```

Comment:
1. using var
2. camelCase 
3. typo 
```suggestion
            var nextOption = Console.ReadLine();
```